### PR TITLE
JENKINS-52393: Adding date and time to log events

### PR DESCRIPTION
**Pull request by danieldanan**
JENKINS-52393: Adding date and time to log events

### CODE REVIEW

File: src/main/java/com/hpe/application/automation/tools/run/PcBuilder.java [hunk 1]
```diff
@@ -196,32 +226,32 @@ public static String getArtifactsDirectoryName() {
         return artifactsDirectoryName;
     }
     
-    public static String getArtifactsResourceName() {
-        
-        return artifactsResourceName;
-    }
+//    public static String getArtifactsResourceName() {
```
Comment (grount) [inline src/main/java/com/hpe/application/automation/tools/run/PcBuilder.java]: Is there any reason for keeping the comments?

File: src/main/java/com/hpe/application/automation/tools/run/PcBuilder.java [hunk 2]
```diff
@@ -107,7 +107,32 @@
...
+    private String _ALMDomain;
+    private String _ALMProject;
+    private String _TestId;
+    private String _TestInstanceId;
+    private String _AutoTestInstanceID;
+    private String _TimeslotDurationHours;
+    private String _TimeslotDurationMinutes;
+    private PostRunAction _PostRunAction;
+    private boolean _VudsMode;
+    private boolean _StatusBySLA;
```
Comment (grount) [inline src/main/java/com/hpe/application/automation/tools/run/PcBuilder.java]: Is there necessity in the unused filed?

**Comment by danieldanan** [comment]
some changes are irrelevant to logs management. canceling them (they will be handled separately).

**Comment by grount** [comment]
Approve for test build

**Comment by grount** [comment]
Approved for test build

**Pull request merged**